docs/content/users/extend/customization-extendibility.md

```diff
@@ -318,7 +318,7 @@ router_http_port: 8080
 
 Extra `config.*.yaml` files are loaded in lexicographic order, so `config.a.yaml` will be overridden by `config.b.yaml`.
 
-Team members may choose to use `config.local.yaml` for local non-committed config changes, for example. `config.local.yaml` is gitignored by default.
+Team members may choose to use `config.local.yaml` for local non-committed config changes, for example. `config.*.local.yaml` files can also be used to modify the order in which the local overrides are loaded. `config.local.yaml` and `config.*.local.yaml` are gitignored by default.
```

Comment:
I guess I didn't understand this had anything to do with overrides? And how does it change order? Looks to me like it's just another config.*.yaml, which happens to be gitignored.

Reply:
@rfay see my comment bellow in the issue queue. I just realize that, that's why I mentioned it on the docs.